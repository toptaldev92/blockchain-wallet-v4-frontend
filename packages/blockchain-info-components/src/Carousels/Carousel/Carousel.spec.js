import { shallow } from 'enzyme'
import React from 'react'
import toJson from 'enzyme-to-json'

import Carousel from './template'

describe('Carousel component', () => {
  it('default renders correctly', () => {
    const component = shallow(
      <Carousel
        height={12}
        chips={false}
        total={12}
        index={4}
        arrows
        handleClick={() => {}}
      />
    )
    const tree = toJson(component)
    expect(tree).toMatchSnapshot()
  })
})
