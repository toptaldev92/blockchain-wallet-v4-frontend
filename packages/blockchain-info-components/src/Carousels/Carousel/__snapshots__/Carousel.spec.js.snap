// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Carousel component default renders correctly 1`] = `
<template__Wrapper>
  <template__Content
    height={12}
    total={12}
  >
    <Styled(Icon)
      left={true}
    />
    <template__Slider
      height={12}
      index={4}
      total={12}
    />
    <Styled(Icon) />
  </template__Content>
</template__Wrapper>
`;
